### DISCUSSION TITLE
Can you guess the artist from these facts?! 🎨 🧑‍🎨

### DISCUSSION THREAD

**Discussion by peymansf2000** [ParanoidUser/yolo#1273] [Galaxy Trivia] [answered]
This artist lived in Mexico City but had European ancestors. They are known for painting self portraits and using bold colours. They also loved butterflies and monkeys. Do you know who they might be?

**Comment by tomerf** [comment, marked as answer]
Sounds like Frida Kahlo is popular around here 😉

**Comment by ParanoidUser** [comment]
![image](https://github.com/ParanoidUser/yolo/assets/5120290/e6ffaa68-b358-4fb3-9349-a8be97a7880f)
